Source/monster.cpp

```diff
@@ -53,7 +53,8 @@ void NewMonsterAnim(int i, AnimStruct *anim, Direction md, AnimationDistribution
 void StartMonsterGotHit(int mid)
 {
 	if (Monsters[mid].MType->mtype != MT_GOLEM) {
-		NewMonsterAnim(mid, &Monsters[mid].MType->Anims[MA_GOTHIT], Monsters[mid]._mdir, gGameLogicStep < GameLogicStep::ProcessMonsters ? AnimationDistributionFlags::ProcessAnimationPending : AnimationDistributionFlags::None);
+		int numSkippedFrames = (gbIsHellfire && Monsters[mid].MType->mtype == MT_DIABLO) ? 4 : 0;
+		NewMonsterAnim(mid, &Monsters[mid].MType->Anims[MA_GOTHIT], Monsters[mid]._mdir, gGameLogicStep < GameLogicStep::ProcessMonsters ? AnimationDistributionFlags::ProcessAnimationPending : AnimationDistributionFlags::None, numSkippedFrames);

```

Comment:
The ternary as the second to last argument here makes the line quite long and a bit hard to read. Consider extracting a variable for that value instead.

Reply:
done 🙂